LightBlue.MultiHost/ViewModel/Role.cs

```diff
@@ -204,33 +204,34 @@ private FifoLog RunnersLog(string runner)
             {
                 return logger;
             }
-            var fifoLog = new FifoLog(50000);
-            Logs.Add(runner, fifoLog);
-            return fifoLog;
+            return EnsureUiThread(() =>
+            {
```

Comment:
When we create the FifoLog on the dispatcher thread (well, technically an STA thread) else Wpf throws a tantrum.